packages/block-library/src/paragraph/style.scss

```diff
@@ -1,46 +1,44 @@
-p {
-	&.is-small-text {
-		font-size: 14px;
-	}
+.is-small-text {
+	font-size: 14px;
+}
 
-	&.is-regular-text {
-		font-size: 16px;
-	}
+.is-regular-text {
+	font-size: 16px;
+}
 
-	&.is-large-text {
-		font-size: 36px;
-	}
+.is-large-text {
+	font-size: 36px;
+}
 
-	&.is-larger-text {
-		font-size: 48px;
-	}
+.is-larger-text {
+	font-size: 48px;
+}
```

Comment:
Is this the right file for these stylesheets? If I remember properly, these are theme options + editor styles? shouldn't we move these styles to the build-int `editor-styles.css` instead?